profilegen/src/res/templates/module_template.template

```diff
@@ -0,0 +1,37 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+require '<%= @module_require %>'
```

Comment:
please leave one line after license as you have in other templates for consistency

Reply:
Done